mod/fbrowser.php

```diff
@@ -56,7 +56,8 @@ function _map_folder1($el){return array(bin2hex($el['album']),$el['album']);};
 				$path[]=array($a->argv[2], $album);
 			}
 
-			$r = q("SELECT `resource-id`, `id`, `filename`, type, min(`scale`) AS `hiq`,max(`scale`) AS `loq`, `desc`
+			$r = q("SELECT `resource-id`, ANY_VALUE(`id`) AS `id`, ANY_VALUE(`filename`) AS `filename`, ANY_VALUE(`type`) AS `type`,
+					min(`scale`) AS `hiq`, max(`scale`) AS `loq`, ANY_VALUE(`desc`) AS `desc`, ANY_VALUE(`created`) AS `created`
```

Comment:
Why had you added "created" here?

Reply:
Because otherwise it is non-standard (and fails in MySQL) to `SORT BY` this field :-)

Reply:
P.S. `ORDER BY created` comes from `$sql_extra2` defined a bit higher up